.github/workflows/docs-pr.yml

```diff
@@ -18,6 +18,7 @@ jobs:
     with:
       init-lenient: false
       init-fail-on-error: true
+      ansible-ref: devel
```

Comment:
This is a bit dangerous since this can break the docs build at any time, but I guess it's OK for now.